### PR TITLE
[IMP] * : improve department's kanban view

### PR DESCRIPTION
**Pull request by mepe-odoo**
* = hr{, _holidays, _skills, _expense, _timesheet, _recruitment, _attendance}

Reorder settings views to be consistent with others modules.

task: 3817650
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_timesheet/views/hr_employee_views.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
         </field>
     </record>
 
-     <record id="view_employee_tree_inherit_timesheet" model="ir.ui.view">
```
Comment (Flotchet) [inline addons/hr_timesheet/views/hr_employee_views.xml:30]: useless diff (not great for history)

**Review by Flotchet — APPROVED** [review]
Just a small comment but otherwise 
Maisse Florent (mafl)

File: addons/hr/models/hr_department.py [hunk 2]
```diff
@@ -135,6 +135,15 @@ def action_plan_from_department(self):
     def get_children_department_ids(self):
         return self.env['hr.department'].search([('id', 'child_of', self.ids)])
 
+    def open_view_dependent_departments(self):
+        return {
+            "type": "ir.actions.act_window",
+            "res_model": "hr.department",
+            "views": [[False, "kanban"], [False, "tree"], [False, "form"]],
+            "domain": [['id', 'in', self.get_children_department_ids().ids]],
+            "name": "Dependent departments",
```
Comment (sofiagvaladze) [inline addons/hr/models/hr_department.py:144]: To be consistent
```suggestion
            "name": "Child Departments",
```

File: addons/hr/models/hr_department.py [hunk 3]
```diff
@@ -135,6 +135,15 @@ def action_plan_from_department(self):
     def get_children_department_ids(self):
         return self.env['hr.department'].search([('id', 'child_of', self.ids)])
 
+    def open_view_dependent_departments(self):
+        return {
```
Comment (sofiagvaladze) [inline addons/hr/models/hr_department.py:139]: naming and also if the action is meant for one record only, it's never a bad idea to put `ensure_one`
```suggestion
    def action_open_view_dependent_departments(self):
        self.ensure_one()
        return {
```

File: addons/hr/models/hr_department.py (hunk 2 above)
Comment (mepe-odoo) [reply addons/hr/models/hr_department.py:144]: Geoffrey ask me to replace Child Departments to Dependant departments :)

Comment (sofiagvaladze) [reply addons/hr/models/hr_department.py:145]: https://www.awesomescreenshot.com/image/47449654?key=c2da6ae5eb06837714b08568775cb2de

Was this before or after :thinking:

Comment (sofiagvaladze) [reply addons/hr/models/hr_department.py:145]: image

Comment (sofiagvaladze) [reply addons/hr/models/hr_department.py:145]: you can check on the task chatter

File: addons/hr/views/hr_department_views.xml [hunk 4]
```diff
@@ -77,17 +77,39 @@
...
+                                <div class="row">
+                                    <div class="o_kanban_card_manage_section o_kanban_manage_view col-6 o_kanban_manage_views">
+                                        <h5 role="menuitem" class="o_kanban_card_manage_title">
+                                            <span>View</span>
+                                        </h5>
+                                        <div role="menuitem">
+                                            <a name="%(act_employee_from_department)d" type="action">Employees</a>
+                                        </div>
+                                        <div role="menuitem">
+                                            <a name="action_open_view_dependent_departments" type="object">Dependent departments</a>
```
Comment (sofiagvaladze) [inline addons/hr/views/hr_department_views.xml:93]: then we change here as well

File: addons/hr_timesheet/views/hr_employee_views.xml [hunk 5]
```diff
@@ -38,23 +38,6 @@
         </field>
     </record>
 
-    <record id="hr_department_view_kanban" model="ir.ui.view">
```
Comment (sofiagvaladze) [inline addons/hr_timesheet/views/hr_employee_views.xml:41]: this is very strange :eyes: 
https://github.com/odoo/odoo/blame/master/addons/hr_timesheet/views/hr_department_views.xml

https://github.com/odoo/odoo/blame/master/addons/hr_timesheet/views/hr_employee_views.xml

and those lines, blame to the same commit as well :exploding_head: 

Probably something went wrong 

https://github.com/odoo/odoo/commit/d01399589f83741a135197052c102365fa8a43b2
here, file was renamed.. 
and this view was added, that was already existing

so, we can remove it

**Comment by sofiagvaladze** [comment]
robodoo r+

**Comment by sofiagvaladze** [comment]
robodoo rebase-ff

**Pull request closed without merging**